assets/images/logo.svg

```diff
@@ -0,0 +1 @@
49.702 8.239c2.228.438 4.151 1.885 5.246 3.948h-.003c1.058 1.83 1.587 4.062 1.587 6.696.026 3.397-.856 6.735-2.548 9.642-1.956 3.406-4.522 5.109-7.695 5.109a8.36 8.36 0 0 1-1.78-.201c-2.228-.437-4.151-1.885-5.246-3.948-1.058-1.862-1.587-4.11-1.587-6.744-.032-3.382.852-6.706 2.551-9.591 1.985-3.406 4.55-5.109 7.695-5.109a8.35 8.35 0 0 1 1.78.198zm-1.39 18.437c1.22-1.131 2.054-2.813 2.502-5.048v.01c.162-.821.243-1.658.239-2.497a9.13 9.13 0 0 0-.577-3.106c-.481-1.301-1.121-2.035-1.921-2.2-1.185-.268-2.324.431-3.416 2.098-.83 1.216-1.418 2.592-1.731 4.046a12.69 12.69 0 0 0-.243 2.5 9.05 9.05 0 0 0 .58 3.096c.479 1.301 1.121 2.033 1.921 2.2s1.685-.201 2.646-1.1z" fill-rule="evenodd"/></g><path d="M121.36 58.229v-6.848h4.891c4.143 0 6.445-2.82 6.445-6.157 0-3.366-2.273-6.186-6.445-6.186h-8.977v19.191h4.086zm4.316-10.444h-4.316v-5.15h4.316c1.611 0 2.848.978 2.848 2.589 0 1.582-1.237 2.561-2.848 2.561zm20.671 10.444v-8.977c0-3.999-2.906-5.265-6.099-5.265-2.187 0-4.374.662-6.071 2.187l1.438 2.561c1.151-1.064 2.532-1.64 4-1.64 1.841 0 3.05.921 3.05 2.331v1.755c-.921-1.093-2.532-1.697-4.403-1.697-2.215 0-4.804 1.208-4.804 4.488 0 3.107 2.589 4.604 4.804 4.604 1.813 0 3.453-.662 4.403-1.813v1.467h3.682zM139.73 56.1c-1.439 0-2.618-.748-2.618-2.072s1.179-2.1 2.618-2.1c1.179 0 2.33.432 2.935 1.237v1.726c-.605.806-1.756 1.208-2.935 1.208zm9.465 7.538c.431.144 1.381.23 1.784.23 2.388-.058 4.229-.748 5.265-3.366l6.531-16.17h-3.913l-3.654 9.667-3.625-9.667h-3.913l5.639 14.069-.518 1.208c-.345.748-.978 1.007-1.927 1.007-.346 0-.835-.086-1.151-.23l-.518 3.251zm35.841-5.409V48.188c0-2.848-1.525-4.201-4.086-4.201-2.129 0-3.971 1.266-4.776 2.503-.489-1.554-1.813-2.503-3.856-2.503-2.129 0-3.941 1.323-4.545 2.129v-1.784h-3.654v13.897h3.654v-9.38c.546-.748 1.611-1.611 2.877-1.611 1.496 0 2.071.921 2.071 2.215v8.775h3.683v-9.38c.518-.748 1.582-1.611 2.877-1.611 1.496 0 2.072.921 2.072 2.215v8.775h3.683zm9.855.345c2.158 0 4.344-.662 5.726-1.928l-1.64-2.417c-.892.863-2.504 1.352-3.683 1.352-2.36 0-3.741-1.41-3.999-3.107h10.329v-.863c0-4.517-2.791-7.624-6.963-7.624-4.201 0-7.193 3.251-7.193 7.279 0 4.46 3.194 7.308 7.423 7.308zm3.222-8.632h-6.847c.144-1.381 1.122-2.963 3.395-2.963 2.417 0 3.337 1.64 3.452 2.963zm18.925 8.287v-9.811c0-2.705-1.468-4.431-4.546-4.431-2.245 0-3.971 1.093-4.863 2.129v-1.784h-3.654v13.897h3.654v-9.38c.633-.806 1.726-1.611 3.165-1.611 1.554 0 2.561.662 2.561 2.59v8.401h3.683zm7.972.345c1.525 0 2.503-.403 3.05-.892l-.777-2.791c-.201.23-.719.432-1.266.432-.805 0-1.266-.662-1.266-1.554v-6.243h2.82v-3.194h-2.82v-3.798h-3.654v3.798h-2.301v3.194h2.301v7.222c0 2.503 1.353 3.827 3.913 3.827zm9.779 0c3.856 0 6.042-1.87 6.042-4.431 0-5.611-8.257-3.884-8.257-5.927 0-.777.863-1.352 2.187-1.352 1.697 0 3.28.748 4.114 1.611l1.467-2.59c-1.381-1.093-3.251-1.899-5.581-1.899-3.683 0-5.697 2.043-5.697 4.373 0 5.467 8.229 3.596 8.229 5.869 0 .863-.749 1.467-2.331 1.467-1.726 0-3.769-.95-4.805-1.928l-1.582 2.647c1.467 1.352 3.798 2.158 6.214 2.158z" fill="#7f54b3"/></svg>
```

Comment:
This SVG file is extracted from from `client/banner/wcpay-logo.js`. It would be great to have a suggestion here to avoid the duplicate. I tried a few ways but they did not work given that: 

1) The Jetpack IDC logo parameter allows only a file path so extracting this file is a requirement. 

2) No simple way to use this file in the `wcpay-logo` React component.

Reply:
It's working very well. Thanks a lot for your suggestion.

Relevant commits: 6cf32a4 and da984dc